OrbitLinuxTracing/PerfEvent.h

```diff
@@ -309,20 +309,25 @@ class UretprobesPerfEvent : public PerfEvent, public AbstractUprobesPerfEvent {
 // perf_event_open event, but we want it to be part of the same hierarchy.
 class MapsPerfEvent : public PerfEvent {
  public:
-  MapsPerfEvent(int32_t pid, uint64_t timestamp, std::string maps)
-      : pid_{pid}, timestamp_{timestamp}, maps_{std::move(maps)} {}
+  MapsPerfEvent(int32_t pid, uint64_t timestamp, const perf_event_mmap_up_to_pgoff& mmap_event,
```

Comment:
As this now now matches a `PERF_RECORD_MMAP` record (unlike before), it should be called `MmapPerfEvent`.

Reply:
done.